File: refactoring-practice-1/src/main/java/dojo/inline_class/Person.java
package dojo.inline_class;

/*
 * Olvasszuk be a Phone osztályt a Person osztályba!
 */
public class Person {

	private String name;
	private Phone officePhone = new Phone();
	
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public Phone getOfficePhone() {
		return officePhone;
	}
}

File: refactoring-practice-1/src/main/typescript/inline_class/person.ts
/*
 * Olvasszuk be a Phone osztályt a Person osztályba!
 */
import {Phone} from "./phone.js";

export class Person {

    private name: string;
    private officePhone: Phone = new Phone();

    public getName() {
        return this.name;
    }

    public setName(name: string) {
        this.name = name;
    }

    public getOfficePhone(): Phone {
        return this.officePhone;
    }
}
